app/db/models/vote.js

```diff
@@ -18,6 +18,8 @@ module.exports = (sequelize, DataTypes) => {
         type: DataTypes.STRING,
         field: 'voter_id'
       },
+      comment: DataTypes.STRING,
```

Comment:
This says `DataTypes.STRING`, but should I be worried that it's not the same as the type specified in the migration file line 13 (where it uses ` DataTypes.TEXT`)?

Reply:
`comment` should be `STRING(280)` everywhere now